tests/components/modbus/test_modbus.py

```diff
@@ -21,6 +22,19 @@
 from .conftest import base_config_test
 
 
+def test_number_helper():
```

Comment:
Why are you testing number (which is located in __init__.py) here ?

if you look in test_init.py you will see it is tested there.